src/sql/orderbook/barn_order_rewards.sql

```diff
@@ -72,7 +72,7 @@ order_surplus AS (
         ON o.uid = oq.order_uid
     WHERE
         ss.block_deadline >= {{start_block}}
-        AND ss.block_deadline <= {{end_block}}
+        AND ss.block_deadline <= {{end_block}} + 100
```

Comment:
The mismatch of `block_deadline` and `block_number` is unfortunate.

If we want to go with that fix, would we not have to decrease the staring value? Something like
```suggestion
        ss.block_deadline >= {{start_block}} - 100
        AND ss.block_deadline <= {{end_block}}
```
We always have `block_deadline > block_number` and (for valid settlements) `block_number > block_deadline - 1001`.

Alternatively, the query for trade_hashes could contain the block deadline via
```sql
        JOIN settlement_scores ss ON s.auction_id = ss.auction_id
    WHERE
        ss.block_deadline > {{start_block}}
        AND ss.block_deadline <= {{end_block}}
```
similarly to how it is done in batch rewards. Then the query itself would be a bit nicer, but potentially there is a mismatch with double counting after a resync.